planning/behavior_velocity_planner/include/scene_module/scene_module_interface.hpp

```diff
@@ -188,18 +197,27 @@ class SceneModuleManagerInterface
     visualization_msgs::msg::MarkerArray debug_marker_array;
     visualization_msgs::msg::MarkerArray virtual_wall_marker_array;
     tier4_planning_msgs::msg::StopReasonArray stop_reason_array;
+    autoware_adapi_v1_msgs::msg::VelocityFactorArray velocity_factor_array;
```

Comment:
[IMO] use alias (https://github.com/autowarefoundation/autoware.universe/pull/2373)

Reply:
I found other lines that doesn't use alias. Since the difference becomes large, it seems better to change it in another PR.

Reply:
OK :+1: